### ISSUE TITLE
Vulnerability roundup 77: python-2.7.17: 4 advisories

### ISSUE THREAD

**Issue by ckauhaus** [NixOS/nixpkgs#73675]
[search](https://search.nix.gsc.io/?q=python&i=fosho&repos=NixOS-nixpkgs), [files](https://github.com/NixOS/nixpkgs/search?utf8=%E2%9C%93&q=python+in%3Apath&type=Code)

* [x] [CVE-2015-5652](https://nvd.nist.gov/vuln/detail/CVE-2015-5652) (nixos-19.09, nixos-unstable)
* [x] [CVE-2017-17522](https://nvd.nist.gov/vuln/detail/CVE-2017-17522) (nixos-19.09, nixos-unstable)
* [ ] [CVE-2017-18207](https://nvd.nist.gov/vuln/detail/CVE-2017-18207) (nixos-19.09, nixos-unstable)
* [ ] [CVE-2019-18348](https://nvd.nist.gov/vuln/detail/CVE-2019-18348) (nixos-19.09, nixos-unstable)

Scanned versions: nixos-19.09: 5f506b95f9f; nixos-unstable: c1966522d7d. May contain false positives.

**Comment by FRidh** [comment]
- [ ] CVE-2015-5652 is for windows and considered longtime behavior. How to mark so it is ignored?
- [ ] CVE-2017-17522  is not considered a (security) bug by upstream and various downstreams.
- [ ] CVE-2017-18207 need to read https://bugs.python.org/issue32056
- [ ] CVE-2019-18348 is a WIP upstream  https://bugs.python.org/issue38576

**Comment by ckauhaus** [comment]
@FRidh No need for special markers. Every vulnerability is reported only once per NixOS release.

**Comment by wamserma** [comment]
@ckauhaus Actually we want to deal with CVE-2015-5652 for good, as it would pop up again each 6 months until python-2.7 is finally retired.

And this is not only python. How about sth. like `meta.cve.wontfix = [ "CVE-2015-5652" "CVE-2017-17522" ] `  in the `.nix` file? Reasons for adding CVEs to this list can be givven in the commit message.

**Comment by redvers** [comment]
master at 2.7.18.
19.09 is EOL

Closing
